terraform/email.tf

```diff
@@ -0,0 +1,35 @@
+#main.tf
+resource "aws_ses_configuration_set" "ses_config" {
+  name = "config_ses"
+  reputation_metrics_enabled = true
+}
+
+resource "aws_ses_domain_identity" "domain_identity" {
+  domain = var.domain_name
+}
+
+resource "aws_ses_domain_dkim" "dkim_identity" {
```

Comment:
resource not necessary already configured in the USDR account bootstrap repo not accessible to volunteers

Confirming that this is already setup in AWS account for CPF
<img width="437" alt="image" src="https://github.com/user-attachments/assets/1a204950-d9bc-4c1f-aee7-830e71240eb3">

Reply:
Done in PR #405